boards/grand_central_m4/Cargo.toml

```diff
@@ -0,0 +1,68 @@
+[package]
+name = "grand_central_m4"
+version = "0.1.0"
+authors = [
+    "Dustin Little <dlittle@toyatech.net>"
+]
+description = "Board Support crate for the Adafruit Grand Central M4 Express"
+keywords = ["no-std", "arm", "cortex-m", "embedded-hal"]
+license = "MIT OR Apache-2.0"
+edition = "2018"
+repository = "https://github.com/atsamd-rs/atsamd"
+readme = "README.md"
+documentation = "https://atsamd-rs.github.io/atsamd/atsamd51g19a/grand_central_m4/"
+
+[dependencies]
+cortex-m = "~0.6"
+embedded-hal = "~0.2.3"
+bitbang-hal = "~0.3"
+apa102-spi = "~0.3"
+smart-leds = "~0.3"
+nb = "~0.1"
+
+[dependencies.cortex-m-rt]
+version = "~0.6.12"
+optional = true
+
+[dependencies.atsamd-hal]
+path = "../../hal"
+version = "~0.9"
+default-features = false
+
+[dependencies.usb-device]
+version = "~0.2"
+optional = true
+
+[dependencies.usbd-serial]
+version = "~0.1"
+optional = true
+
+[dev-dependencies]
+panic-halt = "~0.2"
+panic-semihosting = "~0.5"
+cortex-m-semihosting = "~0.3"
+cortex-m-rtic = "~0.5.1"
+panic_rtt = "~0.2"
+
+[features]
+# ask the HAL to enable atsamd51g19a support
+default = ["rt", "atsamd-hal/samd51p20a", "atsamd-hal/samd51", "atsamd-hal/unproven"]
+rt = ["cortex-m-rt", "atsamd-hal/samd51p20a-rt"]
+unproven = ["atsamd-hal/unproven"]
+usb = ["atsamd-hal/usb", "usb-device", "usbd-serial"]
+use_uart_debug = ["atsamd-hal/use_uart_debug"]
+
+[profile.dev]
+incremental = false
+codegen-units = 1
+debug = true
+lto = true
+
+[profile.release]
+debug = true
```

Comment:
I think this `debug = true` was something I left in the metro_m4 crate by accident, which presumably you copied.

Reply:
yeah I'm still trying to get a handle on how all these pieces fit together. I've removed it in the latest commit